node/src/main.rs

```diff
@@ -302,12 +318,12 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
         .value_of("http-port")
         .unwrap()
         .parse()
-        .expect("invalid GraphQL HTTP server port");
+        .expect("Invalid GraphQL HTTP server port");
```

Comment:
Our convention is for `expect` messages to start with lower-case letters. According to @leoyvens it is more idiomatic.